src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ CHANGELOG
 ---
 
  * Add method `__toString()` to `InputInterface`
+ * Added `OutputWrapperInterface` and `OutputWrapper` to allow modifying your
+   wrapping strategy in `SymfonyStyle` or in other `OutputStyle`. Eg: you can
+   switch off to wrap URLs.
```

Comment:
I'm not convinced we need to open for extensibility here. Fight me if you want to keep it :)

Reply:
I added this because the behavior you need can depend on your console, your language, and other things. I added a really basic solution here, but people may want different behavior, or they want to split the long URL-s on their way. Eg: originally I just wanted to solve the text wrapping but it turned out, URL wrapping is another issue.

Reply:
I agree with Nicolas, we should not make it extensible.